README.md

```diff
@@ -165,6 +165,10 @@ Configure and use Prometheus or a StatsD-interface supporting data store for met
 
 ![Basic Dashboard](./doc/grafana-dashboard.png)
 
+## Logging
+
+To supress [tls handshake error logging](https://cs.opensource.google/go/go/+/master:src/net/http/server.go;l=1933?q=%22TLS%20handshake%20error%20from%20%22&ss=go%2Fgo), set environment variable `SUPPRESSE_TLS_HANDSHAKE_ERROR_LOGGING` to true. See [docker compose](./docker-compose.yml) for example.
```

Comment:
* To `suppress` (typo)...
* `SUPPRESS_TLS_HANDSHAKE_ERROR_LOGGING` (typo, extra `E` in `SUPPRESS`)
* Add backticks around `true` maybe?